homeassistant/components/arcam_fmj/strings.json

```diff
@@ -21,7 +21,7 @@
   },
   "device_automation": {
     "trigger_type": {
-      "turn_on": "{entity_name} was requested to turn on"
+      "turn_on": "[%key:common::device_automation::trigger_type::turned_on%]"
```

Comment:
This is wrong. The device was not turned on. It has a request to be turned on.

Reply:
Thanks!